src/client/commands/submit.rs

```diff
@@ -59,6 +68,7 @@ pub async fn submit_computation(
             stderr: Some(stderr),
             cwd: None,
         },
+        fail_no: opts.max_fails.unwrap(),
```

Comment:
* Please name the attributed and variables that hold this value as "max_fails"
* Do not call "unwrap" on data that came from the user. User should never cause "panic".
* If max fails is not defined, it should behave as the current version